db/external_sst_file_ingestion_job.cc

```diff
@@ -390,34 +390,57 @@ Status ExternalSstFileIngestionJob::AssignLevelForIngestedFile(
     SuperVersion* sv, IngestedFileInfo* file_to_ingest, bool* overlap_with_db) {
   *overlap_with_db = false;
 
-  Arena arena;
-  ReadOptions ro;
-  ro.total_order_seek = true;
-
   Status status;
   int target_level = 0;
   auto* vstorage = cfd_->current()->storage_info();
+
+  // Always go to last level, fail if the file overlaps with existing files.
+  if (ingestion_options_.ingest_behind) {
+    // first check if new files overlaps with any file at the bottommost level
+    int bottom_lvl = cfd_->NumberLevels() - 1;
```

Comment:
So my understanding is, for example, you current num_levels = 3. If you want to ingest_behind, you will reopen the db with num_level = 4 (Get an new empty bottom most level) and  disable_auto_compactions. Then ingest all the files into this new level (level 3)?